libs/chain-signatures/contract/src/tee/tee_state.rs

```diff
@@ -72,6 +68,22 @@ impl TeeState {
         })
     }
 
+    /// Verifies the TEE quote and Docker image
+    pub(crate) fn verify_tee_participant(
+        &mut self,
+        account_id: &AccountId,
+        sign_pk: &PublicKey,
+    ) -> Result<TeeQuoteStatus, Error> {
+        let tee_participant_info = self.tee_participant_info.get(account_id);
+
+        if let Some(tee_participant_info) = tee_participant_info {
+            let tee_participant_info: TeeParticipantInfo = tee_participant_info.clone();
+            self.verify_tee_participant_info(&tee_participant_info, sign_pk)
```

Comment:
Why do we clone the participant info if we later send a reference to the function?